src/knx/data_link_layer.cpp

```diff
@@ -41,11 +41,16 @@ void DataLinkLayer::dataRequest(AckType ack, AddressType addrType, uint16_t dest
     sendTelegram(npdu, ack, destinationAddr, addrType, format, priority, Broadcast);
 }
 
-void DataLinkLayer::systemBroadcastRequest(AckType ack, FrameFormat format, Priority priority, NPDU& npdu)
+void DataLinkLayer::broadcastRequest(AckType ack, FrameFormat format, Priority priority, NPDU& npdu, SystemBroadcast broadcastType)
 {
     // System Broadcast requests will always be transmitted as broadcast with KNX serial number for open media (e.g. RF medium) 
     // See 3.2.5 p.22
-    sendTelegram(npdu, ack, 0, GroupAddress, format, priority, SysBroadcast);
+#if (MEDIUM_TYPE == 5)||(MEDIUM_TYPE == 0)
+    (void)broadcastType; // not used on TP/IP, always normal broadcast
```

Comment:
is the (void)broadcastType; debug code or necessary?

Reply:
I was to lazy to put __attribute__((unused)) in these cases. Shall I use it?
It suppresses compiler warnings about unused vars.

Reply:
I have removed the line for now.
I just realized that there are tons of warnings about unused vars everywhere else. :)
So maybe we should silence this warning for now as it really disturbs the compilation log essentials.